### ISSUE TITLE
Cropping

### ISSUE THREAD

**Issue by jtagcat** [d2phap/ImageGlass#594]
### Is your feature request related to a problem? Please describe.

Looking something to replace the Windows photo viewer (not stable, don't question, believe me),
for a large number of users (easy to learn and use). Currently using IrfanView (what does not look very nice, and a bit unfamiliar for some).

ImageGlass would be great, only crop is missing. Crop is a must have for any quick viewer.
### Describe the solution you'd like
A crop button next to mirror & rotate image buttons. When pressed, makes a same aspect ratio rectangle, what is around 80% the size, making the cropped part darker. Big selectors or large hitboxes for the sides and corners would be nice, as some users struggle to precisely click.
TL;DR:
Add this button
![image](https://user-images.githubusercontent.com/38327267/62405095-f3fb7200-b589-11e9-9267-244427856052.png)
When the button is pressed do this:
![image](https://user-images.githubusercontent.com/38327267/62405409-68cfab80-b58c-11e9-9845-fef8e9f070e2.png)

 - Make it look nicer than a quick paint.net edit
 - Make it easy to use, minimalistic.
 - Accessibility: Crop borders and corners grab range must be quite large, allowing everyone to crop.
### Additional context

Yes, this is a duplicate of #39, directing people away from the project for a (relatively) simple feature is not a solution.

**Comment by Qsefawd** [comment]
oh yes! looking forward for this to be in 7.5.x

**Comment by jtagcat** [comment]
> oh yes! looking forward for this to be in 7.5.x

@Qsefawd Welcome to GitHub issues, GitLab issues, and other issue trackers!
Did you know there's a button for +1 (thumbs up)? Instead of commenting without any relevant information and **sending out notifications to everyone involved**, click the +1 on the original post.

Here's a photo by [Tim van Cleef](https://unsplash.com/@_timvancleef). Have a nice {time of day}!
![tim-van-cleef-1JBOZwuW7sI-unsplash](https://user-images.githubusercontent.com/38327267/77407935-e14edb00-6dae-11ea-8f46-30c78de44743.jpg)

**Comment by d2phap** [comment]
As I already added this to the timeline, so it will be in the next release :)

![image](https://user-images.githubusercontent.com/3154213/77408959-7d9ad100-6df3-11ea-80c6-d94c4386a35e.png)

**Comment by d2phap** [comment]
The first preview version of Cropping tool is now available in the latest build of ImageGlass Moon: https://imageglass.org/moon

### Features
- [x] Copy selected image region to clipboard (Ctrl+Shift+C)
- [x] Crop and Save on the current viewing image
- [x] Crop and Save as a new image file (Ctrl+S)
- [x] Able to crop the image from clipboard (take screenshot -> Ctrl+V to IG -> C to crop)
- [x] Able to crop the image with transparent background (PNG, ...)
- [x] Manually change the crop factors: X, Y, Width, Height
- [x] Draggable / Resizable selection region with 3x3 grid
- [ ] Localization for Cropping tool
- [ ] Tool bar button

### Notes
- Any modification on image may cause loosing meta data and quality
- For multi-page formats, only the viewing page cropping supported
- It's not fully tested and may contain unknown bugs

![image](https://user-images.githubusercontent.com/3154213/77848452-46636000-71f7-11ea-88de-25a1e8282a87.png)

**Comment by d2phap** [comment]
Added Crop button on toolbar, shipped in https://imageglass.org/moon

![image](https://user-images.githubusercontent.com/3154213/79860424-3bba7600-8405-11ea-8b11-e551c0ab6b6b.png)

**Comment by ekottifiok** [comment]
crop generates a very large size image compared to the original 
How do you explain a 37MB picture from a 5MB?

**Comment by jtagcat** [comment]
@ekottifiok don't necro, create a new issue.
